docs/guides/developer/docs/api/usage.md

```diff
@@ -10,16 +10,20 @@ Since the API is versioned, it supports specification of a version identifier as
 
 Header            |Type       | Comment                                                                   
 :-----------------|:----------|:--------------------------------------------------------------------------
-`Accept`          |`String`   | The format is specified to use `application/<version>`, or `application/<version>+<format>` to also specify the required format.
+`Accept`          |`String`   | `application/<version>+<format>` is used to specify the API version and format
```

Comment:
`application/<version>` was never supported so this is an specification error. As it never worked, it can't break any existing clients.